src/main/resources/config/liquibase/changelog/190721_2149_group.xml

```diff
@@ -28,6 +26,15 @@
             <column name="last_modified_by" type="varchar(50)"/>
             <column name="last_modified_date" type="timestamp"/>
         </createTable>
+    </changeSet>
+
+    <changeSet author="georgiana" id="20190822215405">
```

Comment:
Conventia era sa pui indexul urmator al changeset-urilor, dar poti lasa asa.

Reply:
in acest script, ei au inceput numerotarea changeset-urilor de la 3 si ultimul, cel cu load data era 4, si de aceea am pus la final cu 5 si l-am pozitionat inainte de cel cu load data.